### ISSUE TITLE
Enhancement: URL Scheme change

### ISSUE THREAD

**Issue by GastroGeek** [HappenApps/Quiver#1236]
**NOTE**: Whilst I saw a reference to a 'new ticket' here:

https://github.com/HappenApps/Quiver/issues/97#issuecomment-181655522

I didn't find one with quite what I wanted... sorry if I missed it.

Please provide a way in which users can customize what is shown when the URL schema is used to open a note externally - or provide further customization of the schema URL.

Currently, I use Quiver extensively with Alfred... however, its a bit ugly/frustrating that the main app window also opens along with the note I queried for using the URL schema... and that note is in 'editor' mode.

It would be awesome if just the note opened in 'Markdown Preview' mode initially...

Maybe via something like:

```
quiver:///notes/{noteId}?mode=editor
quiver:///notes/{noteId}?mode=preview
quiver:///notes/{noteId}?mode=sidebyside (or split)
```
Something like that, perhaps? Again, most importantly that the main window be hidden and just the desired note be shown?

**Comment by ylian** [comment]
Good idea. I happen to be thinking about enhancing the custom URL scheme support as well. Suggestions are welcome (not limited to displaying notes, but also creating notes).

**Comment by keeprock** [comment]
@GastroGeek could you, please, give a reference, what kind of Alfred extension for Quiver you use? I've tried a few and they all seems broken. Thanks!

**Comment by keeprock** [comment]
@ylian I've used an app that gives a few options upon opening notes using custom URL. You either open in it in a new window, in a main app window or just copy content to a clipboard. Just as ideas for you.

**Comment by GastroGeek** [comment]
@keeprock 

> @GastroGeek could you, please, give a reference, what kind of Alfred extension for Quiver you use? I've tried a few and they all seems broken. Thanks!

I believe It is this one that I am currently using (but cannot remember if I modified it)
https://github.com/ajgon/alfred2-quiver-search
